src/lib.rs

```diff
@@ -1176,30 +1176,84 @@ impl Node {
 	}
 
 	/// Close a previously opened channel.
+	///
+	/// Will attempt to close a channel coopertively. If this fails, users might need to resort to
+	/// [`Node::force_close_channel`].
 	pub fn close_channel(
 		&self, user_channel_id: &UserChannelId, counterparty_node_id: PublicKey,
+	) -> Result<(), Error> {
+		self.close_channel_internal(user_channel_id, counterparty_node_id, false)
+	}
+
+	/// Force-close a previously opened channel.
+	///
+	/// Will force-close the channel, potentially broadcasting our latest state. Note that in
+	/// contrast to cooperative closure, force-closing will have the channel funds time-locked,
+	/// i.e., they will only be available after the counterparty had time to contest our claim.
+	/// Force-closing channels also more costly in terms of on-chain fees. So cooperative closure
+	/// should always be preferred (and tried first).
+	///
+	/// Broadcasting the closing transactions will be omitted for Anchor channels if we trust the
+	/// counterparty to broadcast for us (see [`AnchorChannelsConfig::trusted_peers_no_reserve`]
+	/// for more information).
+	pub fn force_close_channel(
```

Comment:
Update the commit message in terms of new API method rather than `force` flag detail.